Shopping List/ShoppingListCollectionViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ShoppingListCollectionViewController.swift
+//  Shopping List
+//
+//  Created by Alex on 5/3/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
+
+class ShoppingListCollectionViewController: UICollectionViewController {
+    
+    let shoppingListController = ShoppingListController()
+     let reuseIdentifier = "ItemCell"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Register cell classes
+        self.collectionView!.register(UICollectionViewCell.self, forCellWithReuseIdentifier: reuseIdentifier)
+
+        // Do any additional setup after loading the view.
+    }
+
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using [segue destinationViewController].
+        // Pass the selected object to the new view controller.
+    }
+    */
+
+    // MARK: UICollectionViewDataSource
+
+    override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of items
+        return shoppingListController.shoppingList.count
+    }
+
+    override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: reuseIdentifier, for: indexPath) as! ShoppingListCollectionViewCell
+        let shoppingItemSelected = shoppingListController.shoppingList[indexPath.item]
+                
+        cell.itemNameLbl.text = shoppingItemSelected.itemName
+        cell.addedNotAddedLbl.text = shoppingItemSelected.addedToList ? "Added" : "Not Added"
+        cell.imageView.image = UIImage(named: shoppingItemSelected.imageName)
+        
+        cell.itemSelection = shoppingItemSelected
+    
+        return cell
+    }
+
+    override func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        let selectedItem = shoppingListController.shoppingList[indexPath.item]
+        shoppingListController.toggleAdd(shoppingListItem: selectedItem)
+        collectionView.reloadData()
+    }
+    
+    // MARK: UICollectionViewDelegate
```

Comment:
It's good practice to clean up your code when you finish - removing unused cocoa boilerplate that Xcode puts in files when they are created is the first thing I get rid of. Also, You have a stray ViewController class that should be deleted as well, along with any errant print statements or notes to yourself that may not be relevant anymore. A nice trick to make that easier is to search for ```print(``` in Xcode's find navigator (for print statements) and using ```//FIXME: - comments about what to fix or notes to self here``` This makes them show up in the bar above your editor. 
![image](https://user-images.githubusercontent.com/44586402/57175582-5e7a0600-6e13-11e9-9d10-1eeae57a19c7.png)